Add Latvian translations

andreyv opened the pull request:
This is the initial version of the Latvian translations file.

<!-- Reviewable:start -->

---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cms-dev/cms/672)
<!-- Reviewable:end -->

karliss commented on cms/locale/lv/LC_MESSAGES/cms.po:
```diff
@@ -0,0 +1,1065 @@
+msgstr "Kompilācija ir nesekmīga"
+
+msgid "Your submission did not compile correctly."
+msgstr "Jūsu iesūtījums nav nokompilēts."
```
This sounds like compilation wasn't attempted instead of compilation failure.

karliss commented:
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 270 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwA9PHNPJ_S2AK2rFf:-KVwA9PHNPJ_S2AK2rFg:bjrqnso) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L270)):*
> ```pot
> msgid "Every minute "
> msgid_plural "Every %(gen_interval)g minutes "
> msgstr[0] "Katru minūti "
> ```

Incorrect for 21.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 846 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwEydAYUb_mGXd9S0d:-KVwEydAYUb_mGXd9S0e:b-9obraz) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L846)):*
> ```pot
> 
> msgid "File (text or PDF)"
> msgstr "Datnes (teksta vai PDF)"
> ```

Shouldn't this be in singular?

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 903 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwFxfxxXQJhgRO5kpF:-KVwFxfxxXQJhgRO5kpG:bxfb14e) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L903)):*
> ```pot
> 
> msgid "Played"
> msgstr "Spēlēts"
> ```

This is used in context of tokens. "Izmantots" might be more appropriate.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672)*
<!-- Sent from Reviewable.io -->

karliss commented:
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 297 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwMA03uGkmJIYRX9NG:-KVwMA04Pf2wDgq78Nxy:bjt7sm0) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L297)):*
> ```pot
> #, python-format
> msgid "You don't get other %(type_pl)s."
> msgstr "Jums netiek piešķirts %(type_pl)s."
> ```

piešķirts žetoni

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672)*
<!-- Sent from Reviewable.io -->

karliss commented:
Review status: 0 of 1 files reviewed at latest revision, 6 unresolved discussions.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 785 at r1](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVxqE2lD9_iklg1ZQYF:-KVxqE2lD9_iklg1ZQYG:b-3vm9fi) ([raw file](https://github.com/cms-dev/cms/blob/05ca9a2bdc1a9ea44f37b039a8221356758c9126/cms/locale/lv/LC_MESSAGES/cms.po#L785)):*
> ```pot
> 
> msgid "Start!"
> msgstr "Starts!"
> ```

This  is on a button and might mean to start not a message to participant.  Needs to be investigated. "Sākt!" might be better.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672)*
<!-- Sent from Reviewable.io -->

andreyv commented:
Review status: 0 of 1 files reviewed at latest revision, 6 unresolved discussions.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 32 at r1](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVuVtzbn-0FxPa9-orO-r1-32:-KW4TRuBdYZSRGC5NUTC:b-2c5gct) ([raw file](https://github.com/cms-dev/cms/blob/05ca9a2bdc1a9ea44f37b039a8221356758c9126/cms/locale/lv/LC_MESSAGES/cms.po#L32)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> This sounds like compilation wasn't attempted instead of compilation failure.
</details>
Should be better now.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 785 at r1](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVxqE2lD9_iklg1ZQYF:-KW4TkN-hcgebZlfjIuK:bt2vr1b) ([raw file](https://github.com/cms-dev/cms/blob/05ca9a2bdc1a9ea44f37b039a8221356758c9126/cms/locale/lv/LC_MESSAGES/cms.po#L785)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> This  is on a button and might mean to start not a message to participant.  Needs to be investigated. "Sākt!" might be better.
</details>
Fixed.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 270 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwA9PHNPJ_S2AK2rFf:-KW4TfrPnoonwQVgyu6d:b-v177zp) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L270)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> Incorrect for 21.
</details>
Fixed, also in some other places.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 297 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwMA03uGkmJIYRX9NG:-KW4Tj1p4w3xdGXXmw0-:bt2vr1b) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L297)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> piešķirts žetoni
</details>
Fixed.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 846 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwEydAYUb_mGXd9S0d:-KW4TlsK04P32sC7wpDX:bt2vr1b) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L846)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> Shouldn't this be in singular?
</details>
Fixed.

---

*[cms/locale/lv/LC_MESSAGES/cms.po, line 903 at r3](https://reviewable.io:443/reviews/cms-dev/cms/672#-KVwFxfxxXQJhgRO5kpF:-KW4TnsA0LkXE1msN6BZ:bt2vr1b) ([raw file](https://github.com/cms-dev/cms/blob/80a01375ca7c502bdb2a726cc1b28184e882257f/cms/locale/lv/LC_MESSAGES/cms.po#L903)):*
<details><summary><i>Previously, karliss wrote…</i></summary>
> This is used in context of tokens. "Izmantots" might be more appropriate.
</details>
Fixed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672)*
<!-- Sent from Reviewable.io -->

karliss commented:
The translation looks reasonable. Someone from core team should probably check if there are no technical problems.

stefano-maggiolo commented:
Reviewed 1 of 1 files at r1, 1 of 1 files at r12.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672)*
<!-- Sent from Reviewable.io -->

stefano-maggiolo commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/672#-:-KY3Z18flIegs2jBLnVy:bnfp4nl)*
<!-- Sent from Reviewable.io -->

stefano-maggiolo commented:
Merged!
